angular.json

```diff
@@ -83,7 +83,6 @@
               "node_modules/bootstrap/dist/css/bootstrap.min.css",
               "node_modules/patternfly/dist/css/patternfly.min.css",
               "node_modules/patternfly/dist/css/patternfly-additions.min.css",
-              "node_modules/patternfly-ng/dist/css/patternfly-ng.min.css",
```

Comment:
It's not necessary, but maybe it'd be good to add the CSS file for Toastr here so that when debugging a test in the browser, the notification would appear in the correct location.